lib/fuzzily/model.rb

```diff
@@ -39,7 +38,7 @@ def _matches_for_trigrams(trigrams)
 
       def _add_fuzzy_scopes
         scope :for_model, lambda { |model|
-          where(owner_type: model.kind_of?(Class) ? model.name : model)
+          where(owner_type: model.demodulize)
```

Comment:
Does this not need to handle both the `Class` and `String` cases anymore?

Reply:
no, there are no usages calling this with the class in this repo or scriptdash. i'm going to adjust this slightly so it reads a little better.